### DISCUSSION TITLE
Do you have plans to make your documents multilingual?

### DISCUSSION THREAD

**Discussion by nitaking** [blitz-js/blitz#1106] [Ideas] [answered]
I think we'll need multilingual support for docment in the future in order to have an efficient development experience in blitz.

On the other hand, I think it's still too early to make this support in the beta state where the documentation is unstable.

It's an idea that probably won't be rejected, but where do you envision doing this?
(I'd like to be proactive in dealing with Japanese! (And I'm sure many of you think so!

**Comment by flybayer** [comment, marked as answer]
Yes!! We have an open issue for this on the website repo: https://github.com/blitz-js/blitzjs.com/issues/20

You can comment there if you want to help with Japanese!

**Reply by nitaking** [reply]
Oops! I'm sorry!
Thanks for the heads up!

**Reply by flybayer** [reply]
No problem!!
